### DISCUSSION TITLE
`registry_package` event doesn't trigger action

### DISCUSSION THREAD

**Discussion by GabrielCousin** [community/community#26927] [Actions]
<p>Hi there!</p>
<p>I managed to create two actions on 2 free private repository:</p>
<ul>
<li>the first one triggers an action when <code>registry_package</code> event is fired (when publishing the package on GitHub Package Registry), do a CURL to dispatch an event on the 2nd repo</li>
<li>the second one triggers an action when <code>repository_dispatch</code> event is fired which opens a PR</li>
</ul>
<p>It was working fine / as expected when using it in my repos (it was a test) - and I couldn’t make it work in repos of my company. I was wondering if there was some limitations based on billing plans when you’re part of an organization? Or maybe I’m missing something?</p>
<p>Here is the first action which is neither starting nor listed :slight_smile:<br>
Thanks!</p>
<pre><code class="lang-auto">on:
  registry_package
name: Notify
jobs:
  Release:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - uses: actions/setup-node@v1
      with:
        node-version: '10.x'
    - name:
      env:
        GITHUB_TOKEN: ${{ secrets.TOKEN_NAME }}
      run: |
        curl -s -X POST -H "Authorization: token ${GITHUB_TOKEN}" --data "{\"event_type\": \"on-demand-upgrade\", \"client_payload\": {\"key\": \"value\"}}" https://api.github.com/repos/ORG_NAME/REPO_NAME/dispatches
</code></pre>

**Comment by Yanjingzhu** [comment]
<p>Hi <a class="mention" href="/gabrielcousin">@GabrielCousin</a>,<br>
Could you please confirm that the <code>registry_package</code> event was happened? Was the new package shown in <a href="https://github.com/%7Borg%7D/%7Brepo%7D/packages" rel="nofollow noopener">https://github.com/{org}/{repo}/packages</a> page?<br>
How did you publish package to GitHub Package Registry? Did you use your PAT or use workflow scope GITHUB_TOKEN?</p>

**Comment by GabrielCousin** [comment]
<p>Hi,<br>
Thanks for replying!</p>
<p>The <code>registry_package</code> event was fired in my fired and then triggering the other action as expected in the repos I own. But it’s not working on the repos of my company.</p>
<p>Both packages were being published through GitHub actions and are listed correctly in the <a href="https://github.com/ORG_OR_USER/REPO_NAME/packages" rel="nofollow noopener">https://github.com/ORG_OR_USER/REPO_NAME/packages</a></p>
<p>My repo is using a PAT whereas the one from my company is using the “default” <code>GITHUB_TOKEN</code></p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Event that created by GITHUB_TOKEN could not trigger another workflow.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a7ec31df07de09d36e68ace99684311ab1172dac.png" data-download-href="/uploads/short-url/nXvMwlXjMBrwexffy13ESUnOC72.png?dl=1" title="GHT" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181100823-c4c33e84-1b36-43b2-b5b5-ee460cd0be4a.png" alt="GHT" data-base62-sha1="nXvMwlXjMBrwexffy13ESUnOC72" width="538" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a7ec31df07de09d36e68ace99684311ab1172dac_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">GHT</span><span class="informations">784×728 40.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
This screenshot is from document: <a href="https://help.github.com/en/actions/reference/events-that-trigger-workflows#about-workflow-events" rel="nofollow noopener">https://help.github.com/en/actions/reference/events-that-trigger-workflows#about-workflow-events</a><br>
Please consider to use PAT instead of GITHUB_TOKEN.

**Comment by GabrielCousin** [comment]
<p>:man_facepalming: Thanks a lot for helping us!</p>
